tests/test_api.rs

```diff
@@ -277,6 +277,26 @@ async fn can_list_metadata() {
         entries[0].get("time").unwrap().get("end").unwrap(),
         1503074938.2232
     );
+    assert_eq!(
+        entries[0]
+            .get("custom")
+            .unwrap()
+            .as_object()
+            .unwrap()
+            .get("orderly")
+            .unwrap()
+            .as_object()
+            .unwrap()
+            .get("artefacts")
+            .unwrap()
+            .as_array()
+            .unwrap()[3]
+            .get("description")
+            .unwrap()
+            .as_str()
+            .unwrap(),
+        "Projected Coverage for routine immunisation in PINE countries"
+    );
```

Comment:
I know the rest of the file is written in this style and you are just following that, but that is way more verbose than it needs to be.

```rust
assert_eq!(entries[0]["custom"]["orderly"]["artefacts"][3]["description"], "Projected Coverage ...");
```

`serde_json::Value` implements the Index trait for both integers and string keys. If the value isn't of the right kind, the indexing returns a Null variant and the equality assertion would eventually fail. It also has a `PartialEq<&str>` implementation, which means that you shouldn't need the final `.as_str().unwrap()` call.

Alternatively, there is also the `.pointer` method, eg.

```rust
entries[0].pointer("/custom/orderly/artefacts/3/description")
```

Pointer syntax comes from https://www.rfc-editor.org/rfc/rfc6901

https://docs.rs/serde_json/latest/serde_json/value/enum.Value.html#impl-Index%3CI%3E-for-Value
https://docs.rs/serde_json/latest/serde_json/value/enum.Value.html#method.pointer

Reply:
Oh, I hadn't seen pointer syntax before, that's handy!

I will use the index syntax.